.storybook/custom/custom.scss

```diff
@@ -696,3 +696,36 @@ body.sb-show-main {
 [dir="rtl"] .fddocs-margin-end-none {
   margin: 1.5rem;
 }
+
+.fd-user-actions-menu-example {
```

Comment:
Any reason why these rules are custom for storybook and not built in to the component?